docs/rbac/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _rbac-index:
+
+Role-based Access Control
+=========================
+
+Role-based access control (RBAC) **restricts** access to content based on a user's role within an
```

Comment:
Are you talking specifically about "content" here (i think not). Maybe use entities instead. In the context of Pulp, "content" already has quite a distinct meaning.